priv/scripts/elixir.sh

```diff
@@ -80,18 +80,19 @@ function upload_build {
 function update_builds_txt {
   otp_versions=(${@:3})
   date=$(date -u '+%Y-%m-%dT%H:%M:%SZ')
+  build_sha256=$(sha256sum mix.exs | cut -d ' ' -f 1)
 
   aws s3 cp s3://s3.hex.pm/builds/elixir/builds.txt builds.txt || true
   touch builds.txt
 
   sed -i "/^${1} /d" builds.txt
   sed -i "/^${1}-otp-\d\+ /d" builds.txt
 
-  echo -e "${1} ${2} ${date}\n$(cat builds.txt)" > builds.txt
+  echo -e "${1} ${2} ${build_sha256} ${date}\n$(cat builds.txt)" > builds.txt
```

Comment:
this is not ideal, but in order not to break existing tools we cannot add the new field in the middle, it needs to be the last field:

```suggestion
  echo -e "${1} ${2} ${date} ${build_sha256} \n$(cat builds.txt)" > builds.txt
```

wdyt @ericmj?

Reply:
I guess we'll also need to manually update builds.txt with checksums for existing releases.

Reply:
Yes, it should be added to the end.